src/chrome/content/rules/Easel.ly.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+	Not supporting HTTPS or using an invalid certificate:
+		* assets.easel.ly
+		* blog.easel.ly
+		* dev.easel.ly
+		* help.easel.ly
+		* webmail.easel.ly
+
+--><ruleset name="easel.ly">
+
+	<target host="www.easel.ly" />
+
+	<rule from="^http://easel\.ly/"
+		to="https://www.easel.ly/" />
```

Comment:
Adding `easel.ly` as target host will make this obsolete.
^ gets secured and redirected to www, www gets then secured again.